source/roadmap.md

```diff
@@ -147,6 +147,22 @@ Desired features include:
 
 **Expected outcomes:** fpm can generate source code distributions of packages that can be build using CMake and/or Meson.
 
+## Improving Compilers
+
+The aim of this project is to create and maintain an open source Fortran compiler as a community, here is a non-exhaustive list of (independent) features:
+
+* compile all Fortran projects to binaries
+* work interactively (Jupyter notebooks)
+* in the browser
+* works as the foundation for linters, VSCode language integration and documentation generators
+* runs well on modern hardware (CPU, GPU, ...)
+
+There are three open source compilers under development:
+
+* LFortran: https://lfortran.org/
+* Flang: https://releases.llvm.org/15.0.0/tools/flang/docs/index.html
```

Comment:
```suggestion
* Flang: https://flang.llvm.org/docs/
```
A better link? Or `https://flang.llvm.org`, but that redirects to this currently.

Reply:
Thanks! I couldn't find the right link.